tests/__sf_section_julia.test.fish

```diff
@@ -0,0 +1,78 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+set -l LOCAL_JULIA_VERSION 1.0.1
```

Comment:
Since the mock is static, I would personally prefer if we wrote the static version in all tests to make the expected output more explicit.